src/main/resources/reference.conf

```diff
@@ -1208,6 +1208,10 @@ search {
     targets = ${target.outputs.target}
     drugs = ${drug.outputs}
     associations = ${associations.outputs.indirect-by-overall}
+    studies = {
+      format = "parquet"
```

Comment:
why no reference ${common.output-format}?

Reply:
But we ideally will use parquet (unless we cannot for any reason for a specific input). So better to set explicitly.